python/cudf/record.txt

```diff
@@ -0,0 +1,263 @@
+/conda/envs/gdf/lib/python3.7/site-packages/cudf/settings.py
```

Comment:
Do these record.txt files need to be .gitignored? `/conda/envs/gdf` looks like a user-specific path?